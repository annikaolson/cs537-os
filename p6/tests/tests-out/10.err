fusermount: entry for /home/myranda/private/cs537/p4/cs537-os/p6/tests/mnt not found in /etc/mtab
