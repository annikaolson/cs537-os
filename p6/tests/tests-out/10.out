Correct
allocated datablocks on /tmp/myranda/test-disk2: found 0 expected 1
